vent/helpers/meta.py

```diff
@@ -181,20 +182,74 @@ def Jobs():
         pass
 
     # get finished jobs
-    try:
-        d_client = docker.from_env()
-        c = d_client.containers.list(all=True,
-                                     filters={'label': 'vent-plugin'})
-        files = []
-        for container in c:
-            jobs[3] += 1
-            if 'file' in container.attrs['Config']['Labels']:
-                if container.attrs['Config']['Labels']['file'] not in files:
-                    files.append(container.attrs['Config']['Labels']['file'])
-        jobs[2] = len(files) - jobs[0]
-        jobs[3] = jobs[3] - jobs[1]
-    except Exception as e:  # pragma: no cover
-        pass
+    #try:
+    d_client = docker.from_env()
+    c = d_client.containers.list(all=True,
```

Comment:
these 2 lines should be in a try/except - if the client is unreachable it shouldn't just throw an exception in the middle of the app